tests/common.go

```diff
@@ -3501,17 +3525,37 @@ func DeleteBackup(backupName string, backupUID string, orgID string, ctx context
 			break
 		}
 	}
+	if backupObj == nil {
+		return nil, fmt.Errorf("unable to find backup [%s] with uid [%s]", backupName, backupUID)
+	}
 
 	bkpDeleteRequest := &api.BackupDeleteRequest{
-		Name:    backupName,
-		OrgId:   orgID,
-		Uid:     backupUID,
-		Cluster: backupObj.Cluster,
```

Comment:
Why was this removed? Cluster reference is needed for backup deletion in case of csi based backups.

Reply:
With 2.6.0 its optional to pass clusterRef in delete request, Delete thread will internally fetch the cluster ref from backupInfo if are not passing it and deletes the backup . Same applicable for csi as well , when the cluster which backup present is active it will fetch and delete.

Tested with IBM CSI which delete's backup from user and admin .
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/basic-system-test/job/nonpx/job/ibm/job/s3/job/ibm-s3-nativecsi/9/console